python310Packages.fsspec: 2022.3.0 -> 2022.5.0, python310Packages.dask: 2022.02.1 -> 2022.05.0

fabaff opened the pull request:
###### Description of changes
https://github.com/fsspec/filesystem_spec/blob/master/docs/source/changelog.rst#202250

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

fabaff committed:
python310Packages.gcsfs: 2022.3.0 -> 2022.5.0

fabaff committed:
python310Packages.dask-gateway-server: 0.9.0 -> 2022.4.0

fabaff committed:
python310Packages.distributed: 2022.2.1 -> 2022.5.0

fabaff committed:
python310Packages.dask-gateway: 0.9.0 -> 2022.4.0

fabaff committed:
python310Packages.dask: 2022.02.1 -> 2022.05.0

fabaff committed:
python310Packages.locket: 0.2.1 -> 1.0.0

fabaff committed:
python310Packages.s3fs: 2022.2.0 -> 2022.5.0

fabaff committed:
python310Packages.tifffile: 2022.3.25 -> 2022.5.4

fabaff committed:
python310Packages.imageio: 2.16.1 -> 2.19.2

fabaff commented:
- `dask-xgboost` is no longer maintained: #174242

fabaff committed:
python310Packages.pyarrow: disable flaky test

fabaff committed:
python310Packages.dask: 2022.05.0 -> 2022.05.2

fabaff commented:
- `spacy-transformers` failure will be addressed with #175086

fabaff committed:
python310Packages.distributed: 2022.5.0 -> 2022.5.2

fabaff committed:
python310Packages.streamz: disable flaky tests

fabaff committed:
python310Packages.pyarrow: disable flaky test

fabaff committed:
python310Packages.intake: 0.6.4 -> 0.6.5

fabaff commented:
Result of `nixpkgs-review pr 174098` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages marked as broken and skipped:</summary>
  <ul>
    <li>intensity-normalization</li>
    <li>python310Packages.caffe</li>
    <li>python310Packages.caffeWithCuda</li>
    <li>python310Packages.intensity-normalization</li>
    <li>python310Packages.napari</li>
    <li>python310Packages.rl-coach</li>
    <li>python39Packages.intensity-normalization</li>
    <li>python39Packages.rl-coach</li>
  </ul>
</details>
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>python310Packages.sunpy</li>
  </ul>
</details>
<details>
  <summary>161 packages built:</summary>
  <ul>
    <li>dvc</li>
    <li>dvc-with-remotes</li>
    <li>home-assistant-component-tests.environment_canada</li>
    <li>napari (python39Packages.napari)</li>
    <li>parquet-tools</li>
    <li>python310Packages.aplpy</li>
    <li>python310Packages.asdf</li>
    <li>python310Packages.astroquery</li>
    <li>python310Packages.batchgenerators</li>
    <li>python310Packages.boxx</li>
    <li>python310Packages.bpycv</li>
    <li>python310Packages.casa-formats-io</li>
    <li>python310Packages.clifford</li>
    <li>python310Packages.dask</li>
    <li>python310Packages.dask-gateway</li>
    <li>python310Packages.dask-gateway-server</li>
    <li>python310Packages.dask-glm</li>
    <li>python310Packages.dask-image</li>
    <li>python310Packages.dask-jobqueue</li>
    <li>python310Packages.dask-ml</li>
    <li>python310Packages.dask-mpi</li>
    <li>python310Packages.dask-yarn</li>
    <li>python310Packages.datafusion</li>
    <li>python310Packages.datasets</li>
    <li>python310Packages.datashader</li>
    <li>python310Packages.db-dtypes</li>
    <li>python310Packages.devito</li>
    <li>python310Packages.distributed</li>
    <li>python310Packages.env-canada</li>
    <li>python310Packages.fastparquet</li>
    <li>python310Packages.fsspec</li>
    <li>python310Packages.gcsfs</li>
    <li>python310Packages.glymur</li>
    <li>python310Packages.google-cloud-bigquery</li>
    <li>python310Packages.handout</li>
    <li>python310Packages.ibis-framework</li>
    <li>python310Packages.image-match</li>
    <li>python310Packages.imagecorruptions</li>
    <li>python310Packages.imageio</li>
    <li>python310Packages.imgaug</li>
    <li>python310Packages.intake</li>
    <li>python310Packages.intake-parquet</li>
    <li>python310Packages.locket</li>
    <li>python310Packages.mask-rcnn</li>
    <li>python310Packages.moviepy</li>
    <li>python310Packages.napari-console</li>
    <li>python310Packages.napari-svg</li>
    <li>python310Packages.partd</li>
    <li>python310Packages.pims</li>
    <li>python310Packages.pyarrow</li>
    <li>python310Packages.pyfftw</li>
    <li>python310Packages.pyregion</li>
    <li>python310Packages.pytest-astropy</li>
    <li>python310Packages.pytest-astropy-header</li>
    <li>python310Packages.pytorch-lightning</li>
    <li>python310Packages.pyvo</li>
    <li>python310Packages.qiskit</li>
    <li>python310Packages.qiskit-machine-learning</li>
    <li>python310Packages.reproject</li>
    <li>python310Packages.runway-python</li>
    <li>python310Packages.s3fs</li>
    <li>python310Packages.scikitimage</li>
    <li>python310Packages.scmrepo</li>
    <li>python310Packages.slicedimage</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.sparse</li>
    <li>python310Packages.spectral-cube</li>
    <li>python310Packages.streamz</li>
    <li>python310Packages.stumpy</li>
    <li>python310Packages.stytra</li>
    <li>python310Packages.tensorly</li>
    <li>python310Packages.test-tube</li>
    <li>python310Packages.tifffile</li>
    <li>python310Packages.tokenizers</li>
    <li>python310Packages.transformers</li>
    <li>python310Packages.zcs</li>
    <li>python39Packages.apache-beam</li>
    <li>python39Packages.aplpy</li>
    <li>python39Packages.asdf</li>
    <li>python39Packages.astroquery</li>
    <li>python39Packages.batchgenerators</li>
    <li>python39Packages.boxx</li>
    <li>python39Packages.bpycv</li>
    <li>python39Packages.caffe</li>
    <li>python39Packages.caffeWithCuda</li>
    <li>python39Packages.casa-formats-io</li>
    <li>python39Packages.clifford</li>
    <li>python39Packages.coqui-trainer</li>
    <li>python39Packages.dask</li>
    <li>python39Packages.dask-gateway</li>
    <li>python39Packages.dask-gateway-server</li>
    <li>python39Packages.dask-glm</li>
    <li>python39Packages.dask-image</li>
    <li>python39Packages.dask-jobqueue</li>
    <li>python39Packages.dask-ml</li>
    <li>python39Packages.dask-mpi</li>
    <li>python39Packages.dask-yarn</li>
    <li>python39Packages.datafusion</li>
    <li>python39Packages.datasets</li>
    <li>python39Packages.datashader</li>
    <li>python39Packages.db-dtypes</li>
    <li>python39Packages.devito</li>
    <li>python39Packages.distributed</li>
    <li>python39Packages.elegy</li>
    <li>python39Packages.env-canada</li>
    <li>python39Packages.fastparquet</li>
    <li>python39Packages.flax</li>
    <li>python39Packages.fsspec</li>
    <li>python39Packages.gcsfs</li>
    <li>python39Packages.glymur</li>
    <li>python39Packages.google-cloud-bigquery</li>
    <li>python39Packages.handout</li>
    <li>python39Packages.ibis-framework</li>
    <li>python39Packages.image-match</li>
    <li>python39Packages.imagecorruptions</li>
    <li>python39Packages.imageio</li>
    <li>python39Packages.imgaug</li>
    <li>python39Packages.intake</li>
    <li>python39Packages.intake-parquet</li>
    <li>python39Packages.locket</li>
    <li>python39Packages.mask-rcnn</li>
    <li>python39Packages.moviepy</li>
    <li>python39Packages.napari-console</li>
    <li>python39Packages.napari-svg</li>
    <li>python39Packages.optax</li>
    <li>python39Packages.partd</li>
    <li>python39Packages.pims</li>
    <li>python39Packages.pyarrow</li>
    <li>python39Packages.pyfftw</li>
    <li>python39Packages.pyregion</li>
    <li>python39Packages.pytest-astropy</li>
    <li>python39Packages.pytest-astropy-header</li>
    <li>python39Packages.pytorch-lightning</li>
    <li>python39Packages.pyvo</li>
    <li>python39Packages.qiskit</li>
    <li>python39Packages.qiskit-machine-learning</li>
    <li>python39Packages.reproject</li>
    <li>python39Packages.runway-python</li>
    <li>python39Packages.s3fs</li>
    <li>python39Packages.scikitimage</li>
    <li>python39Packages.scmrepo</li>
    <li>python39Packages.slicedimage</li>
    <li>python39Packages.spacy-transformers</li>
    <li>python39Packages.sparse</li>
    <li>python39Packages.spectral-cube</li>
    <li>python39Packages.streamz</li>
    <li>python39Packages.stumpy</li>
    <li>python39Packages.stytra</li>
    <li>python39Packages.sunpy</li>
    <li>python39Packages.tensorflow-datasets</li>
    <li>python39Packages.tensorly</li>
    <li>python39Packages.test-tube</li>
    <li>python39Packages.tifffile</li>
    <li>python39Packages.tokenizers</li>
    <li>python39Packages.transformers</li>
    <li>python39Packages.treex</li>
    <li>python39Packages.zcs</li>
    <li>streamlit</li>
    <li>tartube</li>
    <li>tartube-yt-dlp</li>
    <li>tts</li>
  </ul>
</details>

SuperSandro2000 commented on pkgs/development/python-modules/dask/default.nix:
```diff
@@ -41,48 +44,60 @@ buildPythonPackage rec {
   ];
 
-  dontUseSetuptoolsCheck = true;
+  #dontUseSetuptoolsCheck = true;
```
Whats up with this?

SuperSandro2000 commented on pkgs/development/python-modules/dask-image/default.nix:
```diff
@@ -1,41 +1,52 @@
   ];
 
-  pythonImportsCheck = [ "dask_image" ];
+  prePatch = ''
```
```suggestion
  postPatch = ''
```

fabaff replied:
Addressed in #175774

SuperSandro2000 reviewed (commented):
Can we split this PR up?

fabaff commented:
Result of `nixpkgs-review pr 174098` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages marked as broken and skipped:</summary>
  <ul>
    <li>intensity-normalization</li>
    <li>python310Packages.caffe</li>
    <li>python310Packages.caffeWithCuda</li>
    <li>python310Packages.intensity-normalization</li>
    <li>python310Packages.napari</li>
    <li>python310Packages.rl-coach</li>
    <li>python39Packages.intensity-normalization</li>
    <li>python39Packages.rl-coach</li>
  </ul>
</details>
<details>
  <summary>162 packages built:</summary>
  <ul>
    <li>dvc</li>
    <li>dvc-with-remotes</li>
    <li>home-assistant-component-tests.environment_canada</li>
    <li>napari (python39Packages.napari)</li>
    <li>parquet-tools</li>
    <li>python310Packages.aplpy</li>
    <li>python310Packages.asdf</li>
    <li>python310Packages.astroquery</li>
    <li>python310Packages.batchgenerators</li>
    <li>python310Packages.boxx</li>
    <li>python310Packages.bpycv</li>
    <li>python310Packages.casa-formats-io</li>
    <li>python310Packages.clifford</li>
    <li>python310Packages.dask</li>
    <li>python310Packages.dask-gateway</li>
    <li>python310Packages.dask-gateway-server</li>
    <li>python310Packages.dask-glm</li>
    <li>python310Packages.dask-image</li>
    <li>python310Packages.dask-jobqueue</li>
    <li>python310Packages.dask-ml</li>
    <li>python310Packages.dask-mpi</li>
    <li>python310Packages.dask-yarn</li>
    <li>python310Packages.datafusion</li>
    <li>python310Packages.datasets</li>
    <li>python310Packages.datashader</li>
    <li>python310Packages.db-dtypes</li>
    <li>python310Packages.devito</li>
    <li>python310Packages.distributed</li>
    <li>python310Packages.env-canada</li>
    <li>python310Packages.fastparquet</li>
    <li>python310Packages.fsspec</li>
    <li>python310Packages.gcsfs</li>
    <li>python310Packages.glymur</li>
    <li>python310Packages.google-cloud-bigquery</li>
    <li>python310Packages.handout</li>
    <li>python310Packages.ibis-framework</li>
    <li>python310Packages.image-match</li>
    <li>python310Packages.imagecorruptions</li>
    <li>python310Packages.imageio</li>
    <li>python310Packages.imgaug</li>
    <li>python310Packages.intake</li>
    <li>python310Packages.intake-parquet</li>
    <li>python310Packages.locket</li>
    <li>python310Packages.mask-rcnn</li>
    <li>python310Packages.moviepy</li>
    <li>python310Packages.napari-console</li>
    <li>python310Packages.napari-svg</li>
    <li>python310Packages.partd</li>
    <li>python310Packages.pims</li>
    <li>python310Packages.pyarrow</li>
    <li>python310Packages.pyfftw</li>
    <li>python310Packages.pyregion</li>
    <li>python310Packages.pytest-astropy</li>
    <li>python310Packages.pytest-astropy-header</li>
    <li>python310Packages.pytorch-lightning</li>
    <li>python310Packages.pyvo</li>
    <li>python310Packages.qiskit</li>
    <li>python310Packages.qiskit-machine-learning</li>
    <li>python310Packages.reproject</li>
    <li>python310Packages.runway-python</li>
    <li>python310Packages.s3fs</li>
    <li>python310Packages.scikitimage</li>
    <li>python310Packages.scmrepo</li>
    <li>python310Packages.slicedimage</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.sparse</li>
    <li>python310Packages.spectral-cube</li>
    <li>python310Packages.streamz</li>
    <li>python310Packages.stumpy</li>
    <li>python310Packages.stytra</li>
    <li>python310Packages.sunpy</li>
    <li>python310Packages.tensorly</li>
    <li>python310Packages.test-tube</li>
    <li>python310Packages.tifffile</li>
    <li>python310Packages.tokenizers</li>
    <li>python310Packages.transformers</li>
    <li>python310Packages.zcs</li>
    <li>python39Packages.apache-beam</li>
    <li>python39Packages.aplpy</li>
    <li>python39Packages.asdf</li>
    <li>python39Packages.astroquery</li>
    <li>python39Packages.batchgenerators</li>
    <li>python39Packages.boxx</li>
    <li>python39Packages.bpycv</li>
    <li>python39Packages.caffe</li>
    <li>python39Packages.caffeWithCuda</li>
    <li>python39Packages.casa-formats-io</li>
    <li>python39Packages.clifford</li>
    <li>python39Packages.coqui-trainer</li>
    <li>python39Packages.dask</li>
    <li>python39Packages.dask-gateway</li>
    <li>python39Packages.dask-gateway-server</li>
    <li>python39Packages.dask-glm</li>
    <li>python39Packages.dask-image</li>
    <li>python39Packages.dask-jobqueue</li>
    <li>python39Packages.dask-ml</li>
    <li>python39Packages.dask-mpi</li>
    <li>python39Packages.dask-yarn</li>
    <li>python39Packages.datafusion</li>
    <li>python39Packages.datasets</li>
    <li>python39Packages.datashader</li>
    <li>python39Packages.db-dtypes</li>
    <li>python39Packages.devito</li>
    <li>python39Packages.distributed</li>
    <li>python39Packages.elegy</li>
    <li>python39Packages.env-canada</li>
    <li>python39Packages.fastparquet</li>
    <li>python39Packages.flax</li>
    <li>python39Packages.fsspec</li>
    <li>python39Packages.gcsfs</li>
    <li>python39Packages.glymur</li>
    <li>python39Packages.google-cloud-bigquery</li>
    <li>python39Packages.handout</li>
    <li>python39Packages.ibis-framework</li>
    <li>python39Packages.image-match</li>
    <li>python39Packages.imagecorruptions</li>
    <li>python39Packages.imageio</li>
    <li>python39Packages.imgaug</li>
    <li>python39Packages.intake</li>
    <li>python39Packages.intake-parquet</li>
    <li>python39Packages.locket</li>
    <li>python39Packages.mask-rcnn</li>
    <li>python39Packages.moviepy</li>
    <li>python39Packages.napari-console</li>
    <li>python39Packages.napari-svg</li>
    <li>python39Packages.optax</li>
    <li>python39Packages.partd</li>
    <li>python39Packages.pims</li>
    <li>python39Packages.pyarrow</li>
    <li>python39Packages.pyfftw</li>
    <li>python39Packages.pyregion</li>
    <li>python39Packages.pytest-astropy</li>
    <li>python39Packages.pytest-astropy-header</li>
    <li>python39Packages.pytorch-lightning</li>
    <li>python39Packages.pyvo</li>
    <li>python39Packages.qiskit</li>
    <li>python39Packages.qiskit-machine-learning</li>
    <li>python39Packages.reproject</li>
    <li>python39Packages.runway-python</li>
    <li>python39Packages.s3fs</li>
    <li>python39Packages.scikitimage</li>
    <li>python39Packages.scmrepo</li>
    <li>python39Packages.slicedimage</li>
    <li>python39Packages.spacy-transformers</li>
    <li>python39Packages.sparse</li>
    <li>python39Packages.spectral-cube</li>
    <li>python39Packages.streamz</li>
    <li>python39Packages.stumpy</li>
    <li>python39Packages.stytra</li>
    <li>python39Packages.sunpy</li>
    <li>python39Packages.tensorflow-datasets</li>
    <li>python39Packages.tensorly</li>
    <li>python39Packages.test-tube</li>
    <li>python39Packages.tifffile</li>
    <li>python39Packages.tokenizers</li>
    <li>python39Packages.transformers</li>
    <li>python39Packages.treex</li>
    <li>python39Packages.zcs</li>
    <li>streamlit</li>
    <li>tartube</li>
    <li>tartube-yt-dlp</li>
    <li>tts</li>
  </ul>
</details>

fabaff commented:
> Can we split this PR up?

I already moved some commits to separate PRs. Perhaps one or two commits could be moved out but the rest is pretty much a dep chain.

fabaff committed:
python310Packages.fsspec: 2022.3.0 -> 2022.5.0

fabaff committed:
python310Packages.fastparquet: add optional dependency

fabaff committed:
python310Packages.s3fs: disable on older Python releases

SuperSandro2000 commented:
That should go into python-updates

fabaff committed:
Python310Packages.sparse: disable on older Python releases

fabaff committed:
python310Packages.dask-glm: handle optional dependencies

fabaff committed:
python310Packages.dask-ml: handle optional dependencies

fabaff committed:
python310Packages.dask: 2022.05.0 -> 2022.05.2

fabaff commented:
Result of `nixpkgs-review pr 174098` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages marked as broken and skipped:</summary>
  <ul>
    <li>intensity-normalization</li>
    <li>python310Packages.caffe</li>
    <li>python310Packages.caffeWithCuda</li>
    <li>python310Packages.intensity-normalization</li>
    <li>python310Packages.napari</li>
    <li>python310Packages.rl-coach</li>
    <li>python39Packages.intensity-normalization</li>
    <li>python39Packages.rl-coach</li>
  </ul>
</details>
<details>
  <summary>8 packages failed to build:</summary>
  <ul>
    <li>python310Packages.ibis-framework</li>
    <li>python39Packages.apache-beam</li>
    <li>python39Packages.elegy</li>
    <li>python39Packages.flax</li>
    <li>python39Packages.ibis-framework</li>
    <li>python39Packages.optax</li>
    <li>python39Packages.tensorflow-datasets</li>
    <li>python39Packages.treex</li>
  </ul>
</details>
<details>
  <summary>154 packages built:</summary>
  <ul>
    <li>dvc</li>
    <li>dvc-with-remotes</li>
    <li>home-assistant-component-tests.environment_canada</li>
    <li>napari (python39Packages.napari)</li>
    <li>parquet-tools</li>
    <li>python310Packages.aplpy</li>
    <li>python310Packages.asdf</li>
    <li>python310Packages.astroquery</li>
    <li>python310Packages.batchgenerators</li>
    <li>python310Packages.boxx</li>
    <li>python310Packages.bpycv</li>
    <li>python310Packages.casa-formats-io</li>
    <li>python310Packages.clifford</li>
    <li>python310Packages.dask</li>
    <li>python310Packages.dask-gateway</li>
    <li>python310Packages.dask-gateway-server</li>
    <li>python310Packages.dask-glm</li>
    <li>python310Packages.dask-image</li>
    <li>python310Packages.dask-jobqueue</li>
    <li>python310Packages.dask-ml</li>
    <li>python310Packages.dask-mpi</li>
    <li>python310Packages.dask-yarn</li>
    <li>python310Packages.datafusion</li>
    <li>python310Packages.datasets</li>
    <li>python310Packages.datashader</li>
    <li>python310Packages.db-dtypes</li>
    <li>python310Packages.devito</li>
    <li>python310Packages.distributed</li>
    <li>python310Packages.env-canada</li>
    <li>python310Packages.fastparquet</li>
    <li>python310Packages.fsspec</li>
    <li>python310Packages.gcsfs</li>
    <li>python310Packages.glymur</li>
    <li>python310Packages.google-cloud-bigquery</li>
    <li>python310Packages.handout</li>
    <li>python310Packages.image-match</li>
    <li>python310Packages.imagecorruptions</li>
    <li>python310Packages.imageio</li>
    <li>python310Packages.imgaug</li>
    <li>python310Packages.intake</li>
    <li>python310Packages.intake-parquet</li>
    <li>python310Packages.locket</li>
    <li>python310Packages.mask-rcnn</li>
    <li>python310Packages.moviepy</li>
    <li>python310Packages.napari-console</li>
    <li>python310Packages.napari-svg</li>
    <li>python310Packages.partd</li>
    <li>python310Packages.pims</li>
    <li>python310Packages.pyarrow</li>
    <li>python310Packages.pyfftw</li>
    <li>python310Packages.pyregion</li>
    <li>python310Packages.pytest-astropy</li>
    <li>python310Packages.pytest-astropy-header</li>
    <li>python310Packages.pytorch-lightning</li>
    <li>python310Packages.pyvo</li>
    <li>python310Packages.qiskit</li>
    <li>python310Packages.qiskit-machine-learning</li>
    <li>python310Packages.reproject</li>
    <li>python310Packages.runway-python</li>
    <li>python310Packages.s3fs</li>
    <li>python310Packages.scikitimage</li>
    <li>python310Packages.scmrepo</li>
    <li>python310Packages.slicedimage</li>
    <li>python310Packages.spacy-transformers</li>
    <li>python310Packages.sparse</li>
    <li>python310Packages.spectral-cube</li>
    <li>python310Packages.streamz</li>
    <li>python310Packages.stumpy</li>
    <li>python310Packages.stytra</li>
    <li>python310Packages.sunpy</li>
    <li>python310Packages.tensorly</li>
    <li>python310Packages.test-tube</li>
    <li>python310Packages.tifffile</li>
    <li>python310Packages.tokenizers</li>
    <li>python310Packages.transformers</li>
    <li>python310Packages.zcs</li>
    <li>python39Packages.aplpy</li>
    <li>python39Packages.asdf</li>
    <li>python39Packages.astroquery</li>
    <li>python39Packages.batchgenerators</li>
    <li>python39Packages.boxx</li>
    <li>python39Packages.bpycv</li>
    <li>python39Packages.caffe</li>
    <li>python39Packages.caffeWithCuda</li>
    <li>python39Packages.casa-formats-io</li>
    <li>python39Packages.clifford</li>
    <li>python39Packages.coqui-trainer</li>
    <li>python39Packages.dask</li>
    <li>python39Packages.dask-gateway</li>
    <li>python39Packages.dask-gateway-server</li>
    <li>python39Packages.dask-glm</li>
    <li>python39Packages.dask-image</li>
    <li>python39Packages.dask-jobqueue</li>
    <li>python39Packages.dask-ml</li>
    <li>python39Packages.dask-mpi</li>
    <li>python39Packages.dask-yarn</li>
    <li>python39Packages.datafusion</li>
    <li>python39Packages.datasets</li>
    <li>python39Packages.datashader</li>
    <li>python39Packages.db-dtypes</li>
    <li>python39Packages.devito</li>
    <li>python39Packages.distributed</li>
    <li>python39Packages.env-canada</li>
    <li>python39Packages.fastparquet</li>
    <li>python39Packages.fsspec</li>
    <li>python39Packages.gcsfs</li>
    <li>python39Packages.glymur</li>
    <li>python39Packages.google-cloud-bigquery</li>
    <li>python39Packages.handout</li>
    <li>python39Packages.image-match</li>
    <li>python39Packages.imagecorruptions</li>
    <li>python39Packages.imageio</li>
    <li>python39Packages.imgaug</li>
    <li>python39Packages.intake</li>
    <li>python39Packages.intake-parquet</li>
    <li>python39Packages.locket</li>
    <li>python39Packages.mask-rcnn</li>
    <li>python39Packages.moviepy</li>
    <li>python39Packages.napari-console</li>
    <li>python39Packages.napari-svg</li>
    <li>python39Packages.partd</li>
    <li>python39Packages.pims</li>
    <li>python39Packages.pyarrow</li>
    <li>python39Packages.pyfftw</li>
    <li>python39Packages.pyregion</li>
    <li>python39Packages.pytest-astropy</li>
    <li>python39Packages.pytest-astropy-header</li>
    <li>python39Packages.pytorch-lightning</li>
    <li>python39Packages.pyvo</li>
    <li>python39Packages.qiskit</li>
    <li>python39Packages.qiskit-machine-learning</li>
    <li>python39Packages.reproject</li>
    <li>python39Packages.runway-python</li>
    <li>python39Packages.s3fs</li>
    <li>python39Packages.scikitimage</li>
    <li>python39Packages.scmrepo</li>
    <li>python39Packages.slicedimage</li>
    <li>python39Packages.spacy-transformers</li>
    <li>python39Packages.sparse</li>
    <li>python39Packages.spectral-cube</li>
    <li>python39Packages.streamz</li>
    <li>python39Packages.stumpy</li>
    <li>python39Packages.stytra</li>
    <li>python39Packages.sunpy</li>
    <li>python39Packages.tensorly</li>
    <li>python39Packages.test-tube</li>
    <li>python39Packages.tifffile</li>
    <li>python39Packages.tokenizers</li>
    <li>python39Packages.transformers</li>
    <li>python39Packages.zcs</li>
    <li>streamlit</li>
    <li>tartube</li>
    <li>tartube-yt-dlp</li>
    <li>tts</li>
  </ul>
</details>

SuperSandro2000 commented:
Maybe we should throw this also onto python-updates. I am not sure tbh.

fabaff commented:
> Maybe we should throw this also onto python-updates.

Would make sense as some of the failures seem `jaxlib`-related (missing CPU support while running `nix-review` locally) and could mask other issues.

fabaff commented:
I'm cherry-picking it into #175805.
